app/core/tests/test_admin.py

```diff
@@ -0,0 +1,34 @@
+from django.test import TestCase, Client
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+
+
+class AdminSiteTests(TestCase):
+
+    def setUp(self):
```

Comment:
Why don't you call `super().setUp()`

Reply:
I missed it, for this test it worked but I see now documentation it is advised to do :+1: